src/libraries/sendtohelixhelp.proj

```diff
@@ -200,19 +210,15 @@
     </ItemGroup>
 
     <ItemGroup Condition="'$(TargetOS)' == 'Android'">
-      <!-- We have to define this temp item list because of a bug in MSBuild that would prevent access %(Filename) right away -->
       <_apks Include="$(TestArchiveTestsRoot)**/*.apk" />
       <XHarnessApkToTest Include="@(_apks)">
-        <Targets Condition="'$(TargetArchitecture)' == 'arm'">armeabi-v7a</Targets>
-        <Targets Condition="'$(TargetArchitecture)' == 'arm64'">arm64-v8a</Targets>
-        <Targets Condition="'$(TargetArchitecture)' == 'x64'">x86_64</Targets>
-        <Targets Condition="'$(TargetArchitecture)' == 'x86'">x86</Targets>
         <AndroidPackageName>net.dot.%(Filename)</AndroidPackageName>
-        <AndroidInstrumentationName>net.dot.MonoRunner</AndroidInstrumentationName>
-
-        <TestTimeout>$(_workItemTimeout)</TestTimeout>
+      </XHarnessApkToTest>
+      <_runonlyApks Include="$(TestArchiveRoot)runonly/**/*.apk" />
+      <XHarnessApkToTest Include="@(_runonlyApks)">
+        <AndroidPackageName>net.dot.%(Filename)</AndroidPackageName>
```

Comment:
Could this metadata `<AndroidPackageName>` also be moved up to the ItemDefinitionGroup since its has the same value in both `<XHarnessApkToTest`?

Reply:
It could if `AndroidPackageName` was the same for all the items but it differs on each loop iteration.